internal/crypto/pbkdf.go

```diff
@@ -20,10 +20,10 @@ const (
 	minPbkdfSha256SaltSize = 32 // size in bytes == 128 bits
 
 	// The NIST recommended iterations for PBKDF2 with SHA256 hash is 600,000.
-	pbkdf2Sha256Iterations = 1<<20 - 1<<18 // 786,432
+	pbkdf2Sha256Iterations = 600_000
```

Comment:
Since this is coming from PR https://github.com/kopia/kopia/pull/3810 should this be here?

Reply:
It's coming from #3810 
The target branch should have been that one #3810 to show the actual diff. Now that #3810 has been merged and this one is updated, then that change will go away.